### PR TITLE
Replace Dialogs in preparation for Qt 6.x upgrade

### PR DESCRIPTION
**Pull request by nallath**
https://github.com/Ultimaker/Uranium/pull/778

CURA-8687

### CODE REVIEW

File: resources/qml/Dialogs/AboutDialog.qml [hunk 1]
```diff
@@ -165,11 +165,11 @@ UM.Dialog
         }
     }
 
-    rightButtons: Button
+    rightButtons: Cura.SecondaryButton
     {
         //: Close about dialog button
         id: closeButton
-        text: catalog.i18nc("@action:button","Close");
+        text: catalog.i18nc("@action:button", "Close");
```
Comment (nallath) [inline resources/qml/Dialogs/AboutDialog.qml:172]:
```suggestion
        text: catalog.i18nc("@action:button", "Close")
```

File: resources/qml/Cura.qml [hunk 2]
```diff
@@ -747,20 +743,18 @@ UM.MainWindow
         }
     }
 
-    MessageDialog
+    UM.MessageDialog
     {
         id: packageInstallDialog
         title: catalog.i18nc("@window:title", "Install Package");
```
Comment (nallath) [inline resources/qml/Cura.qml:749]:
```suggestion
        title: catalog.i18nc("@window:title", "Install Package")
```

Boyscouting

File: resources/qml/Dialogs/OpenFilesIncludingProjectsDialog.qml [hunk 3]
```diff
@@ -43,69 +35,31 @@ UM.Dialog
...
-            name: "cura"
-        }
-
-        ButtonGroup
-        {
-            buttons: [cancelButton, importAllAsModelsButton]
-            checkedButton: importAllAsModelsButton
-        }
+        font: UM.Theme.getFont("default")
+        wrapMode: Text.WordWrap
```
Comment (nallath) [inline resources/qml/Dialogs/OpenFilesIncludingProjectsDialog.qml:46]:
```suggestion
```
These values are the default for UM.Label. There is no need to re set them (DRY)

File: resources/qml/Preferences/Materials/MaterialsPage.qml [hunk 4]
```diff
@@ -371,23 +363,21 @@ Item
             var result = Cura.ContainerManager.exportContainer(base.currentItem.root_material_id, selectedNameFilter, fileUrl);
 
             messageDialog.title = catalog.i18nc("@title:window", "Export Material");
```
Comment (nallath) [inline resources/qml/Preferences/Materials/MaterialsPage.qml:365]:
```suggestion
            messageDialog.title = catalog.i18nc("@title:window", "Export Material")
```

Boyscouting

**Comment by nallath** [comment]
ReadonlySpinbox.qml seems to be an empty file. I think something went wrong with the mergeconflict

**Comment by casperlamboo** [comment]
> [](https://github.com/casperlamboo)ReadonlySpinbox.qml seems to be an empty file. I think something went wrong with the mergeconflict
https://github.com/Ultimaker/Cura/pull/11460/commits/86392ab471676bde486f26662bd3e2be1e9c4e17

**Pull request merged**